HW2/src/main/java/com/example/hw2/entity/Author.java

```diff
@@ -0,0 +1,29 @@
+package com.example.hw2.entity;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.util.List;
+
+@Getter
+@Setter
+@Entity
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class Author {
+
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "seq_auth")
+    @SequenceGenerator(
+            name = "seq_auth",
+            allocationSize = 1
+    )
+    private Long id;
```

Comment:
A better way is to use naming convention: `<table_id>` instead of `id`.

The column in DB should be `author_id`, the field name:
```
@Column(name = "author_id")
private Long authorId;
```

*the same for all other entities.